networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -18,6 +18,8 @@ image::527-OpenShift-UDN-isolation-012025.png[The namespace isolation concept in
 [NOTE]
 ====
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
+
+{op-system-base} 8 worker nodes that use `cgroupv1` Linux Control Groups (cgroup) must be reconfigured from `cgroupv1` to `cgroupv2` before creating a user-defined network. For more information, see xref:../../../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-cluster-cgroups-2[Configuring Linux cgroup].
```

Comment:
I think while this note as it applies to `{op-system-base} 8` nodes is good, we need to perhaps make it explicit that Host isolation with UDNs is only supported with cgroupsv2. As a result UDN will not be supported on any worker nodes using cgroupsv1 (irrespective of their `{op-system-base}` version).

Reply:
I would say host isolation is supported with cgroupsv1 too, in a sense that host is not able to reach UDN pods, but the only thing that will not work is kubelet probes for UDN pods. Not sure how deep we want to explain this, just saying switch to cgroupv2 before using udn is also good

Reply:
yep, current version works for me, maybe @abhat has other thoughts?

Reply:
alright, I think rhel8 mention is gone, so hopefully this thread can be resolved